clearwater_cassandra/cassandra_plugin.py

```diff
@@ -229,6 +251,11 @@ def leave_cassandra_cluster(self):
 
         run_command("nodetool decommission", self._sig_namespace)
 
+        # Remove the bootstrapped flag so that we bootstrap correctly
+        # if rejoining the cluster again in future.
+        if os.path.exists(self.BOOTSTRAPPED_FLAG):
```

Comment:
Remove the bootstrapping in progress file too. You probably should also remove these in clearwater-cassandra's postrm.